src/models/v3/users.rs

```diff
@@ -59,11 +59,14 @@ pub struct User {
     pub github_id: Option<u64>,
 }
 
+// TODO: make v2 supported version of this
```

Comment:
TODO- for a different PR?

Reply:
removed it since don't plan on doing it (this route is only used by us)